paasta_tools/cli/cmds/metastatus.py

```diff
@@ -38,10 +38,11 @@ def add_subparser(subparsers):
     )
     status_parser.add_argument(
         '-v', '--verbose',
-        action='store_true',
+        action='count',
         dest="verbose",
-        default=False,
-        help="Print out more output regarding the state of the cluster",
+        default=0,
```

Comment:
Can you add this to the itest to ensure we are invoking this new code path on a real box?